apps/andi/assets/css/extract_steps.scss

```diff
@@ -23,7 +23,19 @@
 }
 
 .example-output {
-    color: green;
+    color: $color-success;
+}
+
+.secret-status-msg {
+    margin-left: 1rem;
+}
+
+.secret-save-success {
```

Comment:
Could we try to still stick with BEM conventions on this? As in, something like `secret__save--success` If we don't want to do that anymore, I understand, but I'd rather we discuss that in A&C and try to not have all sorts of patterns going on.